docs/docs-content/getting-started/additional-capabilities.md

```diff
@@ -0,0 +1,75 @@
+connectivity from these sites back to the cloud. This means you can deploy Kubernetes clusters at scale to a wide range
+of locations, including retail stores, hospitals, restaurants, cruise ships, manufacturing facilities, oil and gas
+sites, and rural areas.
+
+Palette Edge supports both VM and container-based workloads, multiple Kubernetes distributions, and Intel and ARM
+hardware architectures. It is built on top of the open-source project [Kairos](https://kairos.io/), which enables the
+creation and customization of immutable versions of operating systems. Additionally, Palette Edge is designed to scale
+to tens of thousands of locations while enforcing policies locally within each cluster.
+
+Edge clusters are Kubernetes clusters set up on Edge hosts. These hosts can be bare metal or virtual machines located in
+isolated locations. Palette deploys and manages workload clusters at the Edge, and the services continue operating even
+when the connection to the management plane is lost. You can manage Edge clusters locally on-site through the local UI,
+or centrally through the Palette management plane. Palette Edge is able to meet your needs, regardless of the network
+topology your deployments face. Check out the [Palette Edge](../clusters/edge/edge.md) page to learn more about Edge and
+its features.
+
+## Self-Hosted Palette
+
+Palette is available as a fully self-hosted platform offering that you can leverage to manage all your Kubernetes
+clusters. You can install an instance of the Palette management plane in your data center or public cloud provider using
+different hosting models.
+
+This option allows you to meet compliance and legal requirements once you have full control over your workloads,
+determine who has access to them, and ensure that no sensitive data is exposed. You can also upgrade the platform at
+your own pace and configure system settings through the
+[System Console](../enterprise-version/system-management/system-management.md#system-console). Additionally, self-hosted
+Palette enables deployments in air-gapped environments where access to the public cloud is not possible. Explore more on
+the [Self-Hosted Palette](../enterprise-version/enterprise-version.md) page.
+
+## Palette VerteX
+
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes
+workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
+can install in your data center or public cloud provider.
+
+Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into
+its management plane and the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s
+infrastructure and applications. To learn more, check out the [Palette VerteX](../vertex/vertex.md) documentation.
+
```

Comment:
Integrating some of my other text with yours, I came up with something like this:

- Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you can install in your secure data center or public cloud provider.
- Palette is fully proven in operational environments giving it a TRL 9 designation, suitable for use with workloads in production on the high-security side up to IL 5/6/6+ environments. It enables you to deploy and manage the full life-cycle of multiple Kubernetes clusters in government clouds such as AWS and Azure, and in air-gap data centers such as vSphere and Nutanix. VerteX also allows you to operate edge nodes where intermittent connectivity, low bandwidth, or complete disconnection pose a challenge.
- Palette VerteX incorporates validated Federal Information Processing Standards (FIPS) 140-2 cryptographic modules into its management plane and the Kubernetes clusters it deploys, ensuring robust data protection for your organization’s infrastructure and applications. TLS 1.3 is used throughout.
- A suite of scanning tools and CONUS support from a dedicated public sector team round out the offering. To learn more, check out the Palette VerteX documentation.